src/modules/qt/filter_qtblend.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * filter_lightshow.cpp -- animate color to the audio
+ * filter_qtblend.cpp -- Qt composite filter
  * Copyright (C) 2015 Meltytech, LLC
```

Comment:
It would be good to update the copyright date in all of these files that are changed.

Reply:
Done, thanks for looking at it. Since these changes only affect qtblend that is mostly used in Kdenlive, I will merge the changes in a few days if there is no opposition.